### DISCUSSION TITLE
[Q&A] lobe chat不能设置那种stream模式的输出吗？

### DISCUSSION THREAD

**Discussion by xrellana** [lobehub/lobehub#930] [General Question]
刚刚用的时候发现输出非常慢，后来去对比了下其他的工具，发现lobe chat是整个返回了才显示不能stream模式的，请问可以设置吗？

**Comment by arvinxx** [comment]
https://github.com/lobehub/lobe-chat/discussions/531

**Reply by xrellana** [reply]
感谢！！我一会尝试下

**Reply by xrellana** [reply]
刚刚测试了，直连工作正常。如果部署在cloudflare后面的话，stream不工作。谢谢！

**Reply by arvinxx** [reply]
https://github.com/lobehub/lobe-chat/issues/781 这个是相关的不？
